crates/ruff_linter/src/rules/flake8_future_annotations/rules/future_rewritable_type_annotation.rs

```diff
@@ -12,7 +12,7 @@ use crate::checkers::ast::Checker;
 /// PEP 563.
```

Comment:
this should also been changed or?

Reply:
No, this reference is correct.

 Lots of PEP involved here.  :- )